client/app/scripts/utils/array-utils.js

```diff
@@ -26,3 +26,7 @@ export function moveElement(array, from, to) {
   }
   return insertElement(removeElement(array, from), to, array[from]);
 }
+
+export function intersperse(items, value) {
```

Comment:
Would be nice to create a unit test for this function and/or add a comment with an example transform above the function.